### DISCUSSION TITLE
How to get preset variable value list with pyaedt?

### DISCUSSION THREAD

**Discussion by JusMave** [ansys/pyaedt#4213] [Q&A] [answered]
How can I get a list containing a variable's available values from this window?

Here is the screenshot.
![image](https://github.com/ansys/pyaedt/assets/31798536/3d792781-9c08-4df2-9cd7-a0a50921a1b6)

**Comment by mostafaeleswy** [comment]
you can save them as csv file then call it  :
`k_space_grids= np.loadtxt("sweep.csv",delimiter=",", dtype=str)
`

**Comment by gmalinve** [comment, marked as answer]
Hi @JusMave @mostafaeleswy,

There's certainly the way to do that with the `export_to_csv` method.
Suppose you add your parametric sweep as: 
`param = m3d.parametrics.add(your_sweep_var, -20, 0, 20, "LinearStep", parametricname=sweep_name)`
you can export the the parametric setup table as:
`param.export_to_csv(path_to_export)`

Hope this helps,
Please do let me know if I answered your question and close this discussion!

Kind regards,

Giulia
